db/migrate/20180822005555_create_games.rb

```diff
@@ -0,0 +1,14 @@
+class CreateGames < ActiveRecord::Migration[5.0]
+  def change
+    create_table :games do |t|
+      t.integer :user_id
```

Comment:
Is this the user who created the game?

Reply:
(Realized this morning that this was intended to be a comment but I submitted it as a pending review - hopefully it shows up for you now!) Gotcha. I'm trying to think if there is an argument against player_one defaulting to be the player who created the game - unless we allow players to step out of and into games (i.e. allow a game to exist with no players), won't there be redundancy in every game table? Any thoughts there?

Reply:
I thought about that too but tried to stay with the plan as much as possible. It does seem like a redundancy that can be removed. I think we can remove `player_one`.

Reply:
Right, that scenario was definitely way more of a hypothetical reason for having two separate columns. I think our case of having as simple a table as possible, removing either user_id or player_one makes sense.